l10n_br_account_product/models/account_invoice.py

```diff
@@ -756,6 +757,21 @@ def _compute_price(self):
     freight_value = fields.Float(
         'Frete', digits=dp.get_precision('Account'), default=0.00)
     fiscal_comment = fields.Text(u'Observação Fiscal')
+    xped = fields.Char(
+        string=u"Código do Pedido (xPed)",
+        size=15,
+    )
+    nitemped = fields.Char(
+        string=u"Item do Pedido (nItemPed)",
+    )
+
+    @api.onchange("nitemped")
```

Comment:
@danimaribeiro, `onchange` because I don't want the user to have to wait until trying to click "save" to see that the `nItemPed` field is in the wrong format.